tests/platform_tests/daemon/test_psud.py

```diff
@@ -186,8 +193,8 @@ def test_pmon_psud_stop_and_start_status(check_daemon_status, duthosts,
                   "Restarted {} pid should be bigger than {} but it is {}"
                   .format(daemon_name, pre_daemon_pid, post_daemon_pid))
 
-    data_after_restart = wait_data(duthost)
-    verify_data(data_before_restart, data_after_restart)
+    # Wait till DB PSU_INFO key values are restored
+    wait_until(40, 5, 20, get_and_verify_data, duthost, data_before_restart)
```

Comment:
Is this delay of 20sec is required, this tells to sleep first for 20 sec and then check at an interval of 5 secs till 40 sec ?

Reply:
Yes @judyjoseph, 20sec delay is not required actually. Tested it with different chassis after removing the delay. Please find attached the results.

![psud_ch10](https://github.com/sonic-net/sonic-mgmt/assets/114024719/24055a61-1d9e-4de3-8f13-b50f38e5b022)
![psud_ch11](https://github.com/sonic-net/sonic-mgmt/assets/114024719/1eb11b73-8f44-4dc6-8860-af0e8359c9d5)